preview-docs/Webhooks.md

```diff
@@ -0,0 +1,66 @@
+# RFC: Webhooks for schema change notifications
+
+To help you better integrate Apollo Studio into your unique workflow, we are starting some work to offer standard webhooks for certain events. The first of those events will be schema changes - any time we detect a change to the schema for your graph.
+
+We are looking for your feedback!
+
+If you have a related need that is not captured below, or if you see things we might be missing or problematic, please engage with us and ask questions through issues on this repository.
+
+## Plan restrictions
+
+Webhooks are available only to customers on our Enterprise plan.
+
+## How this works
+
+We will create a new integration channel: Webhooks.
+From our Integrations page you can create new Webhooks channels and select which types of events you want sent to your integration endpoint. Today we're launching with support for Schema Change Events.
+
+### Terms
+**Event:** Something that happens in Apollo Studio.
+
+**Event Type:** Events have specific types that denote what happened, including 'schema.change'.
+
+**Event Subscription:** An event subscription is configured to do something when it sees a particular type of event. It is the thing that connects an event to a notification.
+*For example: when a `schema.change` event happens, send a webhook notification to a specified destination.*
+
+**Notification:** The message we send to an external service.
+
+**Webhook:** A type of notification sent via a generic HTTP POST request.
+
+**Channel:** The place where we send the notification. 
+For webhooks this will be a server endpoint URL. 
+For Slack messages this will be a particular Slack channel endpoint.
+
+### Setup
+
+You can configure a Webhook channel at the Graph level. This ability will be restricted to Admins and Contributors in your organization.
+
+You will be asked to specify a name and a destination URL for your webhook channel. You can edit these after initial setup.
+
+Once you've done that you will have the option send a test notification to confirm that your destination service is receiving notifications from us.
+
+When we see a schema change to your Graph we will make an HTTP POST request to the URL specified.
+The body of that request will contain a JSON representation of the changes we identified - fields added, fields removed, fields changed, etc.
+
+```json
+{"sample_body":{"format":"tbd"}}
```

Comment:
I would consider starting a list of the specific information you intend to send -- then it will be easy to add to that list or make PRs to suggest changes to the list. That's probably the most salient bit of info here in terms of whether the payloads will have what people need.

Reply:
Added a JSON example